src/client/checkAttr.js

```diff
@@ -0,0 +1,20 @@
+import createDomElem from './createDomElem'
+import isCustomProp from '../isCustomProp'
+import reactProps from './reactProps'
+
+export default (elem: string | Object, attr: string) => {
```

Comment:
> you can actually use Element instead of Object, right?
  
but we pass here just an object: https://github.com/lttb/is-react-prop/pull/3/files#diff-c4d47be494321bcb7989fbc811500f53R12